zenoh.json5

```
@@ -33,8 +33,10 @@
             // Bucket to which this storage is associated to
             bucket: "zenoh-bucket",
 
-            // If the bucket doesn't exist, create it
-            create_bucket: true,
+            // The storage attempts to create the bucket, but if the bucket already exists and is
+            // owned by you, then with 'reuse_bucket' you can associate that preexisting bucket to
+            // the storage, otherwise it will fail.
+            reuse_bucket: true,
```

Related to how to create the storage, I ended up figuring out this was the cleanest way to handle the issue of "what if the bucket already exists, should we throw an error or not?". We always should try to create the bucket, and if it already exists and is owned by the user, and this flag is set as true, then we should proceed with that existing bucket, I think, otherwise return error.
Shouldn't we first verify if the bucket already exists and is owned by the user before attempting to create the bucket? The answer is that with the amazon api as it is it's best to directly attempt to create the bucket, if it was not the case then the bucket is created and if it was the case then it returns an error we can handle.